CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 57.1.0
 
+* Change all test helpers to use a stub\_ prefix (old names are now aliases)
```

Comment:
Just a nitpick... I'm not sure we need the \ here, I would probably go for a \`stub_\`

Reply:
Erm, I think it's a reserved symbol: https://stackoverflow.com/questions/35827838/how-to-show-underscores-symbol-in-markdown

Reply:
Interesting, have never come across that before, it looks like GitHub flavoured Markdown doesn't follow the "standard" and doesn't italicise letters within a word! Either way, I don't mind that much.

Reply:
That is interesting - I think I'll leave as it is, as it renders something awful if you have a markdown syntax highlighting 🤢 .